client/js/components/statement/listStatements/ListStatements.vue

```diff
@@ -162,7 +162,8 @@
           <dp-flyout data-cy="listStatements:statementActionsMenu">
             <button
               v-if="hasPermission('area_statement_segmentation')"
-              :class="`${(segmentsCount > 0 && segmentsCount !== '-') ? 'is-disabled' : '' } btn--blank o-link--default`"
+              :class="`${(segmentsCount > 0 && segmentsCount !== '-') ? 'is-disabled' : '' }
+              btn--blank o-link--default hover:underline active:underline`"
```

Comment:
I know it was like that before, but please have a look at https://vuejs.org/guide/essentials/class-and-style.
The syntax can be simplified ;-)